src/util/sss_krb5.c

```diff
@@ -1104,7 +1104,6 @@ bool sss_krb5_realm_has_proxy(const char *realm)
 
     kerr = profile_get_values(profile, profile_path, &list);
     if (kerr == PROF_NO_RELATION || kerr == PROF_NO_SECTION) {
-        kerr = 0;
         goto done;
```

Comment:
The value of `res` will be `false` here.
Should it be `true`? It's my assumption based on `kerr = 0`

@jhrozek  is author of this code